revapi-site/src/site/asciidoc/faq.adoc

```diff
@@ -56,3 +56,38 @@ must also be specified to ensure that these extensions are run together and in a
                     </configuration>
                 </plugin>
 ```
+=== How to deal with External Class Exposed In API ?
+This is mainly a warning to ensure that you are exposing class from external dependency on purpose.
+Generally, exposing class from external dependency could be considered as bad design because this could make harder to you to upgrade to more recent version of this dependency. One reason amongst others which makes encapsulation a good practice.
+
+But there is very common use cases where this totally makes sense. The most common one is probably when you have multi-modules maven project. In this case this is usual to have sub-modules exposing classes from another one.
```

Comment:
"But there are use cases where this totally makes sense.  The most common one is probably when you have a multi-module Maven project. In this case it is usual to have some modules exposing classes from another "core" or "api" module."